tests/fixtures/validate_fixtures.py

```
@@ -114,6 +115,31 @@ def expected_ack() -> dict:
     }
 
 
+def expected_fence() -> dict:
+    """ADR-0029 lane, marker, and complete representative result matrix."""
+    publisher = bytes.fromhex("8b8f3a627dd54c408a2b28f71331fe41")
+    lane = lambda sequence: bytes([1]) + publisher + struct.pack("<Q", sequence)
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `lane` as a `def`

(E731)